src/native/external/llvm-libunwind/src/CompactUnwinder.hpp

```diff
@@ -28,34 +28,34 @@ namespace libunwind {
 #if defined(_LIBUNWIND_TARGET_I386)
 /// CompactUnwinder_x86 uses a compact unwind info to virtually "step" (aka
 /// unwind) by modifying a Registers_x86 register set
-template <typename A>
+template <typename A, typename R>
```

Comment:
nit: we can use default parameter to avoid retyping the second argument from all callsites, and revert changes in `UnwindCursor.hpp`.
```suggestion
template <typename A, typename R = Registers_x86>
```
(same for other archs below)

Reply:
Good idea. 👍 I am bit rusty on C++ already.